src/libstd/sync/mpsc/mod.rs

```diff
@@ -949,16 +1145,17 @@ impl<T> Receiver<T> {
     ///
     /// This function will always block the current thread if there is no data
     /// available and it's possible for more data to be sent. Once a message is
-    /// sent to the corresponding [`Sender`], then this receiver will wake up and
-    /// return that message.
+    /// sent to the corresponding [`Sender`] (or [`SyncSender`]), then this
```

Comment:
https://travis-ci.org/rust-lang/rust/jobs/225404572

this line causes a failure on travis.

this is considered a markdown link:

```
[foo] (bar.html)
```

for the same reason this is:

```
[`Sender`] (or [`SyncSender`])
```

to get around this, do something like this:

```
[`Sender`][`Sender`] (or [`SyncSender`])
```